style.css

```diff
@@ -0,0 +1,375 @@
+}
+
+.about {
+  border-top-right-radius: 132px;
+  margin-top: 180px;
+  padding-top: 3%;
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  padding-left: 0.9rem;
+  background-color: var(--main-bg-color);
+}
+
+.social-link {
+  font-weight: 500;
+  color: #7f8cff;
+  font-size: 16px;
+  line-height: 24px;
+  text-transform: uppercase;
+  margin-bottom: 0;
+  font-style: normal;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.small-container .social-link a:hover {
+  background-color: black;
+}
+
+.social-media-link {
+  display: flex;
+  justify-content: space-between;
+  max-width: 180px;
+  padding-inline-start: 0;
+  padding-bottom: 16px;
+  font-size: larger;
+}
+
+/* card info */
+
+.card-info {
+  margin-top: 2rem;
+  display: grid;
+  grid-template: repeat(4, 25%) / auto;
+  padding: 17% 5%;
+  gap: 50px;
+}
+
+.personal-info {
+  margin-top: 4rem;
+}
+
+.card-item {
+  margin: 5% auto;
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  width: 327px;
+  padding: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: var(--main-bg-color);
+}
+
+.title {
+  margin-top: -10px;
+  font-style: normal;
+  color: #172b4d;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.frame-2 {
+  display: flex;
+  padding-inline-start: 0;
+  flex-direction: row;
+  gap: 10px;
+}
+
+.top-title {
+  font-style: normal;
+  color: var(--normal-text-color);
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.info {
+  font-style: normal;
+  color: var(--info-color);
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.description {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: var(--normal-text-color);
+}
+
+.skills {
+  font-weight: 500;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  width: 96%;
+}
+
+.skills > span > img:hover {
+  background-color: var(--light-text-color);
+}
+
+hr.hr-line {
+  border-top: 1px solid #dfe1e6;
+  margin-inline-start: 0;
+  width: 325px;
+}
+
+.content > h1 {
+  color: white;
+}
+
+.text-form {
+  padding: 0 12px;
+}
+
+.add-skills {
+  background: #f7f7f9;
+  display: flex;
+  align-items: center;
+  gap: 15px;
+  width: 305px;
+  height: 66.72px;
+  border-radius: 10px;
+  padding: 3% 6%;
+  margin: 3% 0;
+}
+
+.tags {
+  display: flex;
+  padding: 0;
+  gap: 12px;
+}
+
+.img1 {
+  content: url("./images/Snapshoot Portfolio.png");
+}
+
+.img2 {
+  content: url("./images/Snapshoot Portfolio-2.png");
+}
+
+.img3 {
+  content: url("./images/Snapshoot Portfolio-1.png");
+}
+
+.img4 {
+  content: url("./images/Snapshoot Portfolio-3.png");
+}
+
+.tag-images-1 {
+  display: none;
+}
+
+.more-button,
+.check-button {
+  border: 1px solid #6070ff;
+  border-radius: 10px;
+  width: 124px;
+  height: 48px;
+  padding: 13px;
+  gap: 10px;
```

Comment:
- [ ] Kindly give your `button` more margin as seen in your chosen Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/211861197-0ee6e595-0d36-4fb0-b76c-4c5a84906afc.png)  |  ![image](https://user-images.githubusercontent.com/74814780/211861048-46110787-995c-4f05-a02a-14c2721e8130.png)